### ISSUE TITLE
Issues with protobuf-net

### ISSUE THREAD

**Issue by wind-rider** [tapanila/SharpCaster#24]
Two issues:

- When I reference the nuget package (0.1.5) from a blank UWP app then the protobuf-net library is not copied or referenced. Because of that I couldn't build my app with only the nuget package.
- Besides that I think it is not a good practice to put dll's in git. Is there maybe a Nuget version of the protobuf library available?

Google is [working on one](https://www.nuget.org/packages/Google.Protobuf) but that one seems to only support the proto3 version. Protobuf-net is also a nuget package but it seems there is some issue with the chosen target framework.

**Comment by tapanila** [comment]
The google protobuf library would be perfect and has really great nuget packages. But as you pointed it only supports proto3 and when I tried to use it with Chromecast I noticed that it only supports proto2.

I totally agree about putting compiled dll's into git is not a good practice.
The problem relies on the https://github.com/mgravell/protobuf-net nuget package and build targets.
They removed UWP support on 2.1.0 and the 2.0.0.668 still had a very strange target profile.

This is why I compiled it again myself and included the binaries. I will fork the repo to github and setup a new nuget package so I can get rid of that compiled dll from this project
